src/components/templates/PartyMap/components/Map/Map.tsx

```diff
@@ -253,11 +241,7 @@ export const Map: React.FC<MapProps> = ({
           {roomOverlay}
         </div>
 
-        <UserProfileModal
-          userProfile={selectedUserProfile}
-          show={isUserProfileSelected}
-          onHide={deselectUserProfile}
-        />
+        <UserProfileModal />
```

Comment:
I don't think we need to be rendering `UserProfileModal` in this component anymore do we? It should only be rendered in a single location.